lib/generators/draper/install_generator.rb

```diff
@@ -1,7 +1,7 @@
 module Draper
   module Generators
     class InstallGenerator < Rails::Generators::Base
-      source_root File.expand_path('../templates', __FILE__)
+      source_root File.expand_path("templates", __dir__)
```

Comment:
I really appreciate for your comment. My opinion is very simple. It'll better that be shorter than before. And It'll be easier to understand than before, because directory layers that maintainers imagine are less than before. Of cause if you don't like this change, I'll follow your opinion :)